### DISCUSSION TITLE
Enhancement Request - Implement Send POI to BMW

### DISCUSSION THREAD

**Discussion by ryanbuckner** [bimmerconnected/bimmer_connected#493] [Feature request]
It looks like the API supports sending the vehicle a destination.  One thing that is missing from the MyBMW Remote app is the ability to schedule trips or make them reoccurring.  This is an important feature because most of us make the exact same trips at the exact same times of day. 

If possible, I'd like the ability to send a Trip to the car, but understanding that may not be possible, sending a destination would be great too. 

This feature could be used to schedule your trip to school or work every morning, and home in the evening.

**Comment by rikroe** [comment]
While we can't send trips (or multiple POIs at once), we can send single POIs via the [CLI](https://bimmer-connected.readthedocs.io/en/latest/cli.html#sendpoi) or via [HomeAssistant](https://www.home-assistant.io/integrations/bmw_connected_drive/#send-a-point-of-interest-to-your-vehicle).
